shell/platform/darwin/macos/framework/Headers/FlutterPluginRegistrarMacOS.h

```diff
@@ -36,11 +36,30 @@ FLUTTER_DARWIN_EXPORT
 @property(nonnull, readonly) id<FlutterTextureRegistry> textures;
 
 /**
- * The default view displaying Flutter content.
+ * The view displaying Flutter content.
  *
- * This method may return |nil|, for instance in a headless environment.
+ * Currently Flutter only supports one view, and this is the view.
  *
- * The default view is a special view operated by single-view APIs.
+ * Flutter plans to support multiple views in the future. Although single-view
```

Comment:
I feel a bit uncomfortable about making sooo many concrete promises about how this will work in the future without having an actual implementation to back this up. I would keep this a little lighter and stick to what this does today, maybe with just a one-sentence hint that at some point flutter will support multiple views and at some point after that people will eventually have to migrate away from this API after the proper deprecation period. As we implement more stuff, the doc can evolve then.

Spending 90% of the doc on describing what this may do at some unknown point in the future seems odd.

Reply:
Yeah, that's why my initial PR included nothing but the current status of this property. 
```
/**
 * The view displaying Flutter content.
 *
 * This method may return |nil|, for instance in a headless environment.
 */
```
What do you think of this one?

Reply:
I like that more than making promises about the future 😝

Reply:
I still think we should nudge users away from this property though. If we want to give minimal explanation, perhaps we could reconsider this suggestion: https://github.com/flutter/engine/pull/43364#discussion_r1248322959

```
The implicit view that displays Flutter content.

This property is provided for backwards compatibility for apps
that assume a single view. This will eventually be replaced by
a multi-view API variant.

This method may return |nil|, for instance in a headless environment
or for apps that do not assume a single view.
```

Reply:
@loic-sharma Currently there's nowhere no nudge users to. It's not about being minimal, but that everything it talks about is not there yet. Imagine if you're a user, you see this comment that announces an upcoming deprecation, what can you do? You can not migrate, you just keep worrying until the official support is here?

Reply:
And to be honest the doc here is not the point of this PR at all. I only wanted to rename the variables because my other two PRs will add more similar constants (as you have seen) and I don't want them to use the legacy name. Then I found this doc is outdated so I fixed it too.

Reply:
Loïc's suggestion in https://github.com/flutter/engine/pull/43364#discussion_r1256172796 also seems fine. It doesn't make any promises about HOW this may work in the future.

Reply:
Ok, I'll use that one.